### PR TITLE
added arguments for the bash script, fixed some tab-space issues and …

### PR DESCRIPTION
**Pull request by taozhaolib**
…modified the target of registries untar

### CODE REVIEW

File: dspace_metadata_import.sh [hunk 1]
```diff
@@ -3,31 +3,67 @@
...
+
+
+EXISTING_DSPACE_DATABASE_NAME_CHANGED=dspace_changed_from_existing_temp
+NEW_DSPACE_DATABASE_NAME=dspace_new_temp
+
+while getopts o:p:q:r:s:t: option
+  do
+    case "${option}"
+    in
+      o) PATH_TO_REGISTRIES_FROM_OLD_DSPACE=${OPTARG};;
```
Comment (lo5an) [inline dspace_metadata_import.sh]: Not a huge deal, but I'd either use positional arguments or make these command line switches more mnemonic. Just using `-p -q -r -s -t` seems a little odd.

File: dspace_metadata_import.sh [hunk 2]
```diff
@@ -49,7 +85,7 @@ EOF
 
 echo "#####  Import metadata into the new database  #####"
 
-sudo -u postgres psql -U $DSPACE_DATABASE_OWNER -h localhost $NEW_DSPACE_DATABASE_NAME < $PATH_TO_OLD_DSPACE_DATABASE_DATA
+sudo -u postgres psql -U $DSPACE_DATABASE_OWNER -h $DATABASE_HOSTNAME $NEW_DSPACE_DATABASE_NAME < $PATH_TO_OLD_DSPACE_DATABASE_DATA
```
Comment (lo5an) [inline dspace_metadata_import.sh]: I don't think this will handle authentication with a remote RDS database, which we'll need for prod.

**Review by lo5an — COMMENTED** [review]
Great cleanup! Still needs work before it'll do authentication with remote RDS dbs.

File: dspace_metadata_import.sh (hunk 2 above)
Comment (taozhaolib) [reply dspace_metadata_import.sh]: Need help from this. Do you have an example of this? Thanks.

Comment (lo5an) [reply dspace_metadata_import.sh]: You can find some examples in https://github.com/OULibraries/dspace_vagrant
That work never made it to production, but the shell scripts are in pretty good shape.

The main thing is to switch to password-based authentication, which means that you'll probably want to set and use the `PGPASSWORD` environment variable in your scripts.

**Review by lo5an — COMMENTED** [review]
This looks like it has the needed auth changes. Let me know if we need to do anything to support testing against the RDS dbs. 

One thing I'm not seeing is how `dspace_metadata_import.sh` gets installed on the remote server. Should the script get moved in to `files` and copied somewhere under `/opt` as part of this role?

**Comment by taozhaolib** [comment]
Good Points. Will put it into the files.

**Pull request merged**